app/views/directives/bind-service.html

```diff
@@ -0,0 +1,39 @@
+<div class="wizard-pf-body">
+  <div class="wizard-pf-steps">
+    <ul class="wizard-pf-steps-indicator">
+      <li class="wizard-pf-step" ng-class="{
+        active: step.selected,
+        visited: step.visited && !step.selected
+      }" ng-repeat="step in ctrl.steps" data-tabgroup="{{$index}}">
+        <a ng-click="ctrl.stepClick(step)"><span class="wizard-pf-step-number">{{$index + 1}}</span><span class="wizard-pf-step-title">{{step.label}}</span></a>
+      </li>
+    </ul>
+  </div>
+  <div class="container-fluid wizard-pf-main mar-left-none">
+    <div ng-include="ctrl.currentStep.view" class="wizard-pf-contents"></div>
+  </div>
+  <div class="config-bottom modal-footer wizard-pf-footer">
+    <button
+        type="button"
+        class="btn btn-default wizard-pf-cancel wizard-pf-dismiss"
+        ng-class="{'btn-cancel': ctrl.steps.length > 2}"
+        ng-disabled="ctrl.currentStep.id === 'results'"
+        ng-click="ctrl.closeWizard()">
+      Cancel
+    </button>
+    <button
+        type="button"
+        class="btn btn-primary wizard-pf-next"
+        ng-disabled="ctrl.currentStep.id === 'results'"
```

Comment:
Should we hide the button on the last page instead of just disable? Currently what we do for order service.